### PR TITLE
add solution to stars problem

### PR DESCRIPTION
**Pull request by vinasxz**
- [DEMO LINK](https://vinasxz.github.io/layout_stars/)
- [TEST REPORT LINK](https://vinasxz.github.io/layout_stars/report/html_report/)

Checklist:
- [x] Yellow stars are added with container modifier + pseudo-selector (NO extra classes)
- [x] Each BEM block has its own separate file
- [x] All `Typical Mistakes` from `BEM` lesson theory are checked.
- [x] Code follows all the [Code Style Rules ❗️](./checklist.md)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -12,7 +12,50 @@
...
+
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
```
Comment (pedro-ruas) [inline src/index.html:23]: You missed empty lines between sibling blocks (between every stars--N and the next one), you can check [Code Style Rules](https://github.com/mate-academy/layout_stars/blob/master/checklist.md).

**Review by pedro-ruas — CHANGES_REQUESTED** [review]
Your work is very good, the only issue is the missing empty lines between siblings  (between every stars--N and the next one), you can check [Code Style Rules](https://github.com/mate-academy/layout_stars/blob/master/checklist.md).

File: src/index.html [hunk 2]
```diff
@@ -12,7 +12,100 @@
       href="./style.css"
     />
   </head>
+
   <body>
-    <h1>Stars</h1>
+
+
```
Comment (jv-aquino) [inline src/index.html:18]: some unnecessary space here

File: src/index.html [hunk 3]
```diff
@@ -12,7 +12,100 @@
...
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+
+    </div>
+
+
```
Comment (jv-aquino) [inline src/index.html:108]: and here

**Review by jv-aquino — APPROVED** [review]
Amazing job! Clean code overall and just the right amount of CSS to make everything work!

**Pull request closed without merging**